partiql-plan/src/main/kotlin/org/partiql/plan/v1/operator/rex/Rex.kt

```diff
@@ -0,0 +1,15 @@
+package org.partiql.plan.v1.operator.rex
+
+import org.partiql.types.PType
+
+/**
+ * TODO DOCUMENTATION
+ */
+public interface Rex {
+
+    public fun getType(): PType
+
+    public fun getOperands(): List<Rex>
```

Comment:
What's the point of this? I'm thinking of `RexStruct` right now.

Reply:
For the tree, but could be removed until needed..

> `getInputs()` and `getOperands()` make these data structures a more generic tree, but these could be removed and added later (if needed)